From 74f61ead6d425960359ab1ddcd87badd4a26ff62 Mon Sep 17 00:00:00 2001
From: Colin McCabe <cmccabe@cloudera.com>
Date: Mon, 14 Mar 2016 15:11:04 -0700
Subject: [PATCH 1318/2848] Revert "HDFS-9947. Block#toString should not
 output information from derived classes
 (cmccabe)"

This reverts commit 47898ac61eef57bef92cd8dee95f76cb2f1071ff.

Change-Id: I4233c5c1b72fc041c2e7daa7f57dc4b404fd0657
---
 .../org/apache/hadoop/hdfs/protocol/Block.java     |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/protocol/Block.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/protocol/Block.java
index 0b0c42a..f86854d 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/protocol/Block.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/protocol/Block.java
@@ -158,9 +158,9 @@ public void setGenerationStamp(long stamp) {
    */
   @Override
   public String toString() {
-    sb.append(BLOCK_FILE_PREFIX).
-       append(b.blockId).append("_").
-       append(b.generationStamp);
+    StringBuilder sb = new StringBuilder();
+    this.appendStringTo(sb);
+    return sb.toString();
   }
 
   public void appendStringTo(StringBuilder sb) {
-- 
1.7.9.5

